Sources/Datadog/RUM/RUMMonitor/Scopes/Utils/RUMViewUpdatesSampler.swift

```diff
@@ -0,0 +1,52 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal protocol RUMViewUpdatesSamplerType {
```

Comment:
Can we rename this to `throttler` instead of sampler? it can be confused with sampler we apply to session.

Reply:
Good point 👍. Renamed to "throttler" with `accept(event:)` API:
```swift
internal protocol RUMViewUpdatesThrottlerType {
    func accept(event: RUMViewEvent) -> Bool
}
```